src/diffusers/image_processor.py

```diff
@@ -54,7 +54,7 @@ def is_valid_image(image) -> bool:
         `bool`:
             `True` if the input is a valid image, `False` otherwise.
     """
-    return isinstance(image, PIL.Image.Image) or isinstance(image, (np.ndarray, torch.Tensor)) and image.ndim in (2, 3)
```

Comment:
but if it is a PIL.Image, `isinstance(image, PIL.Image.Image)` will evaluate to `True` and whole thing will just return `True` no? 
can you show an example that this fails?
```python
isinstance(image, PIL.Image.Image) or isinstance(image, (np.ndarray, torch.Tensor)) and image.ndim in (2, 3)
```

Reply:
It's true... I tried it locally and you were right.
I seem to have misunderstood the cause and made a false assumption. I will withdraw the PR. I apologize.